src/classification/random_forests.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import confusion_matrix, accuracy_score, precision_score, recall_score, f1_score
+
+class MachineLearning():
+
+    def __init__(self):
+        
+        print("Loading dataset ...")
+        
+        self.flow_dataset = pd.read_csv('/content/drive/MyDrive/COMS6200/NF-UNSW-NB15.csv')
+
+        self.flow_dataset.iloc[:, 0] = self.flow_dataset.iloc[:, 0].str.replace('.', '')
+        self.flow_dataset.iloc[:, 2] = self.flow_dataset.iloc[:, 2].str.replace('.', '')
+
+    def flow_training(self, n_estimators=10, criterion='entropy', max_depth=None):
+        print("Flow Training with n_estimators={}, criterion={}, max_depth={} ...".format(n_estimators, criterion, max_depth))
+        
+        X_flow = self.flow_dataset.iloc[:, :-1].values
+        X_flow = X_flow.astype('float64')
+
+        y_flow = self.flow_dataset.iloc[:, -1].values
+
+        X_flow_train, X_flow_test, y_flow_train, y_flow_test = train_test_split(X_flow, y_flow, test_size=0.25, random_state=0)
```

Comment:
Can you please try it with a different test size like 0.3?

Reply:
Will do it after checking the accuracy with it.